module/VuFind/tests/unit-tests/src/VuFindTest/Controller/Plugin/ResultScrollerTest.php

```diff
@@ -423,7 +423,7 @@ public function __construct($mockManager, $testResults)
      *
      * @return \VuFind\Search\Base\Results
      */
-    protected function restoreLastSearch()
+    protected function restoreLastSearch(): ?\VuFind\Search\Base\Results
```

Comment:
Should `@return` above be updated to match?

Reply:
Done.